backend/src/models/tutorAvailability.js

```diff
@@ -81,4 +81,27 @@ export class tutorAvailability {
         }
 
     }
+
+    async deleteAvailability(tutorId, startDate, endDate) {
+        const client = await pgPool.connect();
+        try {
+            return new Promise((resolve, reject) => {
+                con.query(
+                    'CALL deleteTutorAvailabilityByWeek(?, ?, ?)',
```

Comment:
Nitpicking: Should we be using dollar notation for consistency? We are using it in `tutorRegistration.js`

Ex. `CALL deleteTutorAvailabilityByWeek($1, $2, $3)`?